diff -up glslang-10-11.0.0/glslang/CMakeLists.txt.omv~ glslang-10-11.0.0/glslang/CMakeLists.txt
diff -up glslang-10-11.0.0/hlsl/CMakeLists.txt.omv~ glslang-10-11.0.0/hlsl/CMakeLists.txt
--- glslang-10-11.0.0/hlsl/CMakeLists.txt.omv~	2020-12-27 18:36:14.778525680 +0100
+++ glslang-10-11.0.0/hlsl/CMakeLists.txt	2020-12-27 18:37:30.324645676 +0100
@@ -40,6 +40,7 @@
 add_library(HLSL ${LIB_TYPE} "stub.cpp")
 set_property(TARGET HLSL PROPERTY FOLDER hlsl)
 set_property(TARGET HLSL PROPERTY POSITION_INDEPENDENT_CODE ON)
+set_target_properties(HLSL PROPERTIES VERSION "${GLSLANG_VERSION}" SOVERSION "${GLSLANG_VERSION_MAJOR}")
 
 if(WIN32 AND BUILD_SHARED_LIBS)
     set_target_properties(HLSL PROPERTIES PREFIX "")
diff -up glslang-10-11.0.0/SPIRV/CMakeLists.txt.omv~ glslang-10-11.0.0/SPIRV/CMakeLists.txt
--- glslang-10-11.0.0/SPIRV/CMakeLists.txt.omv~	2020-12-27 18:37:46.782885443 +0100
+++ glslang-10-11.0.0/SPIRV/CMakeLists.txt	2020-12-27 18:38:30.678518151 +0100
@@ -71,6 +71,7 @@ set(SPVREMAP_HEADERS
 add_library(SPIRV ${LIB_TYPE} ${SOURCES} ${HEADERS})
 set_property(TARGET SPIRV PROPERTY FOLDER glslang)
 set_property(TARGET SPIRV PROPERTY POSITION_INDEPENDENT_CODE ON)
+set_target_properties(SPIRV PROPERTIES VERSION "${GLSLANG_VERSION}" SOVERSION "${GLSLANG_VERSION_MAJOR}")
 target_include_directories(SPIRV PUBLIC
     $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/..>
     $<INSTALL_INTERFACE:${CMAKE_INSTALL_INCLUDEDIR}>)
@@ -84,6 +85,7 @@ if (ENABLE_SPVREMAPPER)
     add_library(SPVRemapper ${LIB_TYPE} ${SPVREMAP_SOURCES} ${SPVREMAP_HEADERS})
     set_property(TARGET SPVRemapper PROPERTY FOLDER glslang)
     set_property(TARGET SPVRemapper PROPERTY POSITION_INDEPENDENT_CODE ON)
+    set_target_properties(SPVRemapper PROPERTIES VERSION "${GLSLANG_VERSION}" SOVERSION "${GLSLANG_VERSION_MAJOR}")
 endif()
 
 if(WIN32 AND BUILD_SHARED_LIBS)
