diff -up glslang-11.11.0/CMakeLists.txt.4~ glslang-11.11.0/CMakeLists.txt
--- glslang-11.11.0/CMakeLists.txt.4~	2022-08-26 20:36:16.194369970 +0200
+++ glslang-11.11.0/CMakeLists.txt	2022-08-26 20:36:16.275370604 +0200
@@ -380,7 +380,7 @@ if(ENABLE_GLSLANG_INSTALL)
         include("@PACKAGE_PATH_EXPORT_TARGETS@")
     ]=])
     
-    set(PATH_EXPORT_TARGETS "${CMAKE_INSTALL_DATADIR}/${PROJECT_NAME}/glslang-targets.cmake")
+    set(PATH_EXPORT_TARGETS "${CMAKE_INSTALL_LIBDIR}/cmake/${PROJECT_NAME}/glslang-targets.cmake")
     configure_package_config_file(
         "${CMAKE_CURRENT_BINARY_DIR}/glslang-config.cmake.in"
         "${CMAKE_CURRENT_BINARY_DIR}/glslang-config.cmake"
@@ -397,7 +397,7 @@ if(ENABLE_GLSLANG_INSTALL)
     install(
         EXPORT      glslang-targets
         NAMESPACE   "glslang::"
-        DESTINATION "${CMAKE_INSTALL_DATADIR}/${PROJECT_NAME}"
+	DESTINATION "${CMAKE_INSTALL_LIBDIR}/cmake/${PROJECT_NAME}"
     )
     
     install(
@@ -405,6 +405,6 @@ if(ENABLE_GLSLANG_INSTALL)
             "${CMAKE_CURRENT_BINARY_DIR}/glslang-config.cmake"
             "${CMAKE_CURRENT_BINARY_DIR}/glslang-config-version.cmake"
         DESTINATION
-            "${CMAKE_INSTALL_DATADIR}/${PROJECT_NAME}"
+            "${CMAKE_INSTALL_LIBDIR}/cmake/${PROJECT_NAME}"
     )
-endif()
\ No newline at end of file
+endif()
diff -up glslang-11.11.0/glslang/CMakeLists.txt.4~ glslang-11.11.0/glslang/CMakeLists.txt
--- glslang-11.11.0/glslang/CMakeLists.txt.4~	2022-08-12 17:32:10.000000000 +0200
+++ glslang-11.11.0/glslang/CMakeLists.txt	2022-08-26 20:36:16.276370612 +0200
@@ -220,7 +220,7 @@ if(ENABLE_GLSLANG_INSTALL)
             add_library(GenericCodeGen ALIAS glslang::GenericCodeGen)
         endif()
     ")
-    install(FILES "${CMAKE_CURRENT_BINARY_DIR}/glslangTargets.cmake" DESTINATION ${CMAKE_INSTALL_LIBDIR}/cmake)
+    install(FILES "${CMAKE_CURRENT_BINARY_DIR}/glslangTargets.cmake" DESTINATION ${CMAKE_INSTALL_LIBDIR}/cmake/${PROJECT_NAME})
 
     set(ALL_HEADERS
         ${GLSLANG_HEADERS}
diff -up glslang-11.11.0/glslang/OSDependent/Unix/CMakeLists.txt.4~ glslang-11.11.0/glslang/OSDependent/Unix/CMakeLists.txt
--- glslang-11.11.0/glslang/OSDependent/Unix/CMakeLists.txt.4~	2022-08-12 17:32:10.000000000 +0200
+++ glslang-11.11.0/glslang/OSDependent/Unix/CMakeLists.txt	2022-08-26 20:36:16.276370612 +0200
@@ -65,5 +65,5 @@ if(ENABLE_GLSLANG_INSTALL)
 
         add_library(OSDependent ALIAS glslang::OSDependent)
     ")
-    install(FILES "${CMAKE_CURRENT_BINARY_DIR}/OSDependentTargets.cmake" DESTINATION ${CMAKE_INSTALL_LIBDIR}/cmake)
+    install(FILES "${CMAKE_CURRENT_BINARY_DIR}/OSDependentTargets.cmake" DESTINATION ${CMAKE_INSTALL_LIBDIR}/cmake/${PROJECT_NAME})
 endif()
diff -up glslang-11.11.0/glslang/OSDependent/Windows/CMakeLists.txt.4~ glslang-11.11.0/glslang/OSDependent/Windows/CMakeLists.txt
--- glslang-11.11.0/glslang/OSDependent/Windows/CMakeLists.txt.4~	2022-08-12 17:32:10.000000000 +0200
+++ glslang-11.11.0/glslang/OSDependent/Windows/CMakeLists.txt	2022-08-26 20:36:16.276370612 +0200
@@ -60,5 +60,5 @@ if(ENABLE_GLSLANG_INSTALL)
 
         add_library(OSDependent ALIAS glslang::OSDependent)
     ")
-    install(FILES "${CMAKE_CURRENT_BINARY_DIR}/OSDependentTargets.cmake" DESTINATION ${CMAKE_INSTALL_LIBDIR}/cmake)
+    install(FILES "${CMAKE_CURRENT_BINARY_DIR}/OSDependentTargets.cmake" DESTINATION ${CMAKE_INSTALL_LIBDIR}/cmake/${PROJECT_NAME})
 endif()
diff -up glslang-11.11.0/gtests/CMakeLists.txt.4~ glslang-11.11.0/gtests/CMakeLists.txt
--- glslang-11.11.0/gtests/CMakeLists.txt.4~	2022-08-12 17:32:10.000000000 +0200
+++ glslang-11.11.0/gtests/CMakeLists.txt	2022-08-26 20:36:16.276370612 +0200
@@ -81,7 +81,7 @@ if(BUILD_TESTING)
 
                 add_library(glslangtests ALIAS glslang::glslangtests)
             ")
-            install(FILES "${CMAKE_CURRENT_BINARY_DIR}/glslangtestsTargets.cmake" DESTINATION ${CMAKE_INSTALL_LIBDIR}/cmake)
+	    install(FILES "${CMAKE_CURRENT_BINARY_DIR}/glslangtestsTargets.cmake" DESTINATION ${CMAKE_INSTALL_LIBDIR}/cmake/${PROJECT_NAME})
         endif()
 
         set(GLSLANG_TEST_DIRECTORY "${CMAKE_CURRENT_SOURCE_DIR}/../Test")
diff -up glslang-11.11.0/hlsl/CMakeLists.txt.4~ glslang-11.11.0/hlsl/CMakeLists.txt
--- glslang-11.11.0/hlsl/CMakeLists.txt.4~	2022-08-26 20:36:16.251370416 +0200
+++ glslang-11.11.0/hlsl/CMakeLists.txt	2022-08-26 20:36:16.276370612 +0200
@@ -60,5 +60,5 @@ if(ENABLE_GLSLANG_INSTALL)
 
         add_library(HLSL ALIAS glslang::HLSL)
     ")
-    install(FILES "${CMAKE_CURRENT_BINARY_DIR}/HLSLTargets.cmake" DESTINATION ${CMAKE_INSTALL_LIBDIR}/cmake)
+    install(FILES "${CMAKE_CURRENT_BINARY_DIR}/HLSLTargets.cmake" DESTINATION ${CMAKE_INSTALL_LIBDIR}/cmake/${PROJECT_NAME})
 endif()
diff -up glslang-11.11.0/OGLCompilersDLL/CMakeLists.txt.4~ glslang-11.11.0/OGLCompilersDLL/CMakeLists.txt
--- glslang-11.11.0/OGLCompilersDLL/CMakeLists.txt.4~	2022-08-12 17:32:10.000000000 +0200
+++ glslang-11.11.0/OGLCompilersDLL/CMakeLists.txt	2022-08-26 20:36:16.277370620 +0200
@@ -54,6 +54,6 @@ if(ENABLE_GLSLANG_INSTALL)
 
         add_library(OGLCompiler ALIAS glslang::OGLCompiler)
     ")
-    install(FILES "${CMAKE_CURRENT_BINARY_DIR}/OGLCompilerTargets.cmake" DESTINATION ${CMAKE_INSTALL_LIBDIR}/cmake)
+    install(FILES "${CMAKE_CURRENT_BINARY_DIR}/OGLCompilerTargets.cmake" DESTINATION ${CMAKE_INSTALL_LIBDIR}/cmake/${PROJECT_NAME})
 
 endif(ENABLE_GLSLANG_INSTALL)
diff -up glslang-11.11.0/SPIRV/CMakeLists.txt.4~ glslang-11.11.0/SPIRV/CMakeLists.txt
--- glslang-11.11.0/SPIRV/CMakeLists.txt.4~	2022-08-26 20:36:16.251370416 +0200
+++ glslang-11.11.0/SPIRV/CMakeLists.txt	2022-08-26 20:39:59.706096963 +0200
@@ -137,7 +137,7 @@ if(ENABLE_GLSLANG_INSTALL)
 
             add_library(SPVRemapper ALIAS glslang::SPVRemapper)
         ")
-        install(FILES "${CMAKE_CURRENT_BINARY_DIR}/SPVRemapperTargets.cmake" DESTINATION ${CMAKE_INSTALL_LIBDIR}/cmake)
+	install(FILES "${CMAKE_CURRENT_BINARY_DIR}/SPVRemapperTargets.cmake" DESTINATION ${CMAKE_INSTALL_LIBDIR}/cmake/${PROJECT_NAME})
     endif()
 
     file(WRITE "${CMAKE_CURRENT_BINARY_DIR}/SPIRVTargets.cmake" "
@@ -149,7 +149,7 @@ if(ENABLE_GLSLANG_INSTALL)
 
         add_library(SPIRV ALIAS glslang::SPIRV)
     ")
-    install(FILES "${CMAKE_CURRENT_BINARY_DIR}/SPIRVTargets.cmake" DESTINATION ${CMAKE_INSTALL_LIBDIR}/cmake)
+    install(FILES "${CMAKE_CURRENT_BINARY_DIR}/SPIRVTargets.cmake" DESTINATION ${CMAKE_INSTALL_LIBDIR}/cmake/${PROJECT_NAME})
 
     install(FILES ${HEADERS} ${SPVREMAP_HEADERS} DESTINATION ${CMAKE_INSTALL_INCLUDEDIR}/glslang/SPIRV/)
 endif()
diff -up glslang-11.11.0/StandAlone/CMakeLists.txt.4~ glslang-11.11.0/StandAlone/CMakeLists.txt
--- glslang-11.11.0/StandAlone/CMakeLists.txt.4~	2022-08-26 20:36:16.251370416 +0200
+++ glslang-11.11.0/StandAlone/CMakeLists.txt	2022-08-26 20:36:16.277370620 +0200
@@ -119,7 +119,7 @@ if(ENABLE_GLSLANG_INSTALL)
 
         add_library(glslangValidator ALIAS glslang::glslangValidator)
     ")
-    install(FILES "${CMAKE_CURRENT_BINARY_DIR}/glslangValidatorTargets.cmake" DESTINATION ${CMAKE_INSTALL_LIBDIR}/cmake)
+    install(FILES "${CMAKE_CURRENT_BINARY_DIR}/glslangValidatorTargets.cmake" DESTINATION ${CMAKE_INSTALL_LIBDIR}/cmake/${PROJECT_NAME})
 
     if(ENABLE_SPVREMAPPER)
         install(TARGETS spirv-remap EXPORT glslang-targets)
@@ -134,7 +134,7 @@ if(ENABLE_GLSLANG_INSTALL)
 
             add_library(spirv-remap ALIAS glslang::spirv-remap)
         ")
-        install(FILES "${CMAKE_CURRENT_BINARY_DIR}/spirv-remapTargets.cmake" DESTINATION ${CMAKE_INSTALL_LIBDIR}/cmake)
+	install(FILES "${CMAKE_CURRENT_BINARY_DIR}/spirv-remapTargets.cmake" DESTINATION ${CMAKE_INSTALL_LIBDIR}/cmake/${PROJECT_NAME})
     endif()
 
     install(TARGETS glslang-default-resource-limits EXPORT glslang-targets)
@@ -149,6 +149,6 @@ if(ENABLE_GLSLANG_INSTALL)
 
         add_library(glslang-default-resource-limits ALIAS glslang::glslang-default-resource-limits)
     ")
-    install(FILES "${CMAKE_CURRENT_BINARY_DIR}/glslang-default-resource-limitsTargets.cmake" DESTINATION ${CMAKE_INSTALL_LIBDIR}/cmake)
+    install(FILES "${CMAKE_CURRENT_BINARY_DIR}/glslang-default-resource-limitsTargets.cmake" DESTINATION ${CMAKE_INSTALL_LIBDIR}/cmake/${PROJECT_NAME})
 
 endif()
